src/content_api/details_utils.py

```diff
@@ -0,0 +1,246 @@
+# -*- coding: utf-8 -*-
+import json
+from collections import OrderedDict
+
+from bs4 import BeautifulSoup
+from lxml import etree
+from lxml import html
+from pandas.io.json import json_normalize
+
+LOOK = ['title', 'body']
+CHILD_KEYS = ['title', 'description']
+DETAILS_SECTIONS = ['body', 'brand', 'documents', 'final_outcome_detail', 'final_outcome_documents',
+                    'government', 'headers', 'introduction', 'introductory_paragraph',
+                    'licence_overview', 'licence_short_description', 'logo', 'metadata', 'more_information',
+                    'need_to_know',
+                    'other_ways_to_apply', 'summary', 'ways_to_respond', 'what_you_need_to_know', 'will_continue_on',
+                    'parts',
+                    'collection_groups']
+
+
+def is_json(raw_text):
+    try:
+        json_normalize(raw_text).columns.tolist()
```

Comment:
Is this quicker than trying `json.loads(raw_text)`?/

Reply:
I don't think it makes a difference.